scripts/setup.sh

```diff
@@ -75,7 +75,7 @@ download_config() {
 
 fetch_config(){
     if [[ -w ${CONFIG_DIR} ]]; then
-        downloadConfig
+        download_config
```

Comment:
yes absolutely @tonytw1, I didn't want to mix it in the previous PR re bumping library.
It was an omission from a prev setup PR- it used to run a separate module but moved it to a function within the same setup file.
Thanks for doing that 👍